### PR TITLE
Add Bitergia branding to Kibiter 6.8.6

### PR DESCRIPTION
**Pull request by dlumbrer**
This commit has the feature of adding the branding of Bitergia to the Kibiter 6.8.6 version, the same that in 6.1.4, logo, colors and text changed.

This video shows the branding:

Peek 2020-03-09 20-11

### CODE REVIEW

**Comment by GeorgLink** [comment]
It looks amazing 🤩

File: src/ui/public/chrome/_variables.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-$kbnChromeGlobalNavBackground: #005571;
+$kbnChromeGlobalNavBackground: #333;
```
Comment (sduenas) [inline src/ui/public/chrome/_variables.scss]: Shouldn't be better to use 6 digit numbers?

**Comment by dlumbrer** [comment]
PR updated @sduenas , thanks!

File: src/ui/public/assets/favicons/browserconfig.xml [hunk 2]
```diff
@@ -2,8 +2,10 @@
...
-            <square150x150logo src="/mstile-150x150.png"/>
-            <TileColor>#da532c</TileColor>
+            <square70x70logo src="/ms-icon-70x70.png"/>
+            <square150x150logo src="/ms-icon-150x150.png"/>
+            <square310x310logo src="/ms-icon-310x310.png"/>
+            <TileColor>#ffffff</TileColor>
         </tile>
     </msapplication>
-</browserconfig>
+</browserconfig>
```
Comment (sduenas) [inline src/ui/public/assets/favicons/browserconfig.xml]: Add an empty line after this.

**Review by sduenas — CHANGES_REQUESTED** [review]
All looks good but just fix a minor problem.

File: src/ui/public/assets/favicons/browserconfig.xml (hunk 2 above)
Comment (dlumbrer) [reply src/ui/public/assets/favicons/browserconfig.xml]: Done!

**Comment by dlumbrer** [comment]
Updated, thanks @sduenas

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**